src/features/game/scenes/bindings/Bindings.ts

```diff
@@ -67,14 +67,18 @@ class Bindings extends Phaser.Scene {
     });
 
     bindingsContainer.add(
-      bindings.map((binding, index) =>
-        this.createBinding(
+      bindings.map((binding, index) => {
+        return this.createBinding(
           binding.key,
           binding.text,
-          bindingPositions[index][0],
-          bindingPositions[index][1] + BindingConstants.key.yStart
-        )
-      )
+          bindingPositions[index][1] < 450
+            ? bindingPositions[index][0] - 400
+            : bindingPositions[index][0] + 400,
+          bindingPositions[index][1] <= 450
+            ? bindingPositions[index][1] + BindingConstants.key.yStart - 200
+            : bindingPositions[index][1] + BindingConstants.key.yStart - 775
```

Comment:
Would there be any issues with using numbers directly? E.g. have you tested for different screen sizes / browsers?

Reply:
I have tested the website on different browsers (firefox and edge) with different screen size, the binding keys are in the correct positions.

<img width="1280" alt="image" src="https://github.com/source-academy/frontend/assets/138369841/3afd84dd-4a00-45df-8179-35883044955c">

Reply:
Generally, using direct number values for adjusting UI is not recommended given that these values can change for screen sizes, browsers, mobile view, etc. If there is an alternate way to do this, I would be much more open to it! 

But if you guys have sufficiently explored options & think this is the only way, then it should be fine. Do update 👍

Reply:
We tried to use `screenSize` to format the page instead of number values.

Reply:
> Generally, using direct number values for adjusting UI is not recommended given that these values can change for screen sizes, browsers, mobile view, etc. If there is an alternate way to do this, I would be much more open to it!
> 
> But if you guys have sufficiently explored options & think this is the only way, then it should be fine. Do update 👍

I'd like to add on that if you're using numbers directly, it would be good if you defined them in the constants file and imported them into this file instead of using magic numbers, for easy subsequent config.

> We tried to use `screenSize` to format the page instead of number values.

I'd still recommend having some object/value that stores this config in `CommonConstants` instead of calling `screenSize.x` and/or `screenCenter.y - screenSize.y / 4` (or even more complicated formulas, instead of just something like `positionHeightOffset`) every time.

Reply:
Moved the formulas to `BindingsConstants.ts`.